Tests/StreamChatUITests/SnapshotVariant.swift

```diff
@@ -10,6 +10,10 @@ enum SnapshotVariant: String, Hashable, CaseIterable {
     case large
     case extraExtraExtraLarge
     
+    static var lightAndDarkCases: [SnapshotVariant] {
+        [small, large]
+    }
```

Comment:
I think this is useful if we are just interested In testing the dark and light variation.
 Another thing I that bug me is that right now there's no way to know which variant is dark or light. We can change this by renaming the cases to include this, for example: `case largeAndLight, smallAndDark`. 

Or at least, we could change the `rawValue: String` to include the dark and light variant? So it is printed in the snapshot file name? Since if the aim of the snapshot is to check how it looks on dark and light, it could be useful to know which one is dark and light, in case they render the same way 🤔

Reply:
Hmm we discussed in slack that we could use optionSet for the `SnapshotVariants` and make this easily set like `variant: [.small, .dark]` and so, maybe we could create ticket for this and rewrite this a bit

Reply:
Yeah I think the OptionSet is looking to be a better candidate for this. I changed my mind, again 😅

Reply:
Maybe is just better to do it here? This one is pretty small PR, and it will be helpful to change this fast, so the next Audits will get the OptionSet already. WDYT?

Reply:
Sounds good, however we need to carefully check this if this doesn't break anything.

Reply:
Yes, that is a good point. Probably is better to make that PR isolated. It should be a quick one though. I'll remove here the commits changing the SnapshotVariant stuff if we agree on migrating to an OptionSet. CC @VojtaStavik

Reply:
We want to have a unified set of snapshots for every component. So far it seems that we just need to distinguish between views the react to size changes and the ones that don't. I'd suggest creating `userInterfaceStyleCases` as an addition to `allCases` we already have, and use `userInterfaceStyleCases` as the default for the snapshots.